crypto-ffi/bindings/js/CoreCrypto.ts

```diff
@@ -595,21 +600,19 @@ export class CoreCrypto {
      * You will want to use {@link CoreCrypto.addClientsToConversation} afterwards to add clients to this conversation
      *
      * @param conversationId - The conversation ID; You can either make them random or let the backend attribute MLS group IDs
-     * @param configuration.admins - An array of client IDs that will have administrative permissions over the group
+     * @param configuration - configuration of the MLS group
      * @param configuration.ciphersuite - The {@link Ciphersuite} that is chosen to be the group's
-     * @param configuration.keyRotationSpan - The amount of time in milliseconds after which the MLS Keypackages will be rotated
      * @param configuration.externalSenders - Array of Client IDs that are qualified as external senders within the group
```

Comment:
Should maybe add the `configuration.custom` field and redirecting to the `CustomConfiguration` interface there